distribution/examples/monitoring-tracing/opentelemetry/README.md

```
@@ -27,6 +27,13 @@ docker run -it --name jaeger -e COLLECTOR_OTLP_ENABLED=true -p 16686:16686 -p 43
 A span created by Membrane should be visible in the [Jaeger UI](http://localhost:16686).
 ![sample](resources/otel_example.png)
 7. Check the headers printed to the console. You will see headers such as `traceparent`, which indicate the trace and span context involved in the request.
+8. The trace and span IDs are also written into the MDC log context and appear in every log line produced during that request:
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)